draft-wood-icnrg-tlvencap.mkd

```diff
@@ -143,9 +152,14 @@ prefix as used in the key exchagne or it is derived from within CCNxKE from Pref
 The SID is created by the producer and given to the consumer inside CCNxKE.
 
 ~~~
-OuterName := '/' PREFIX '/' SID=sid '/' CHUNK=chunk
+OuterName := ccnx:/PREFIX/SID=sid/CHUNK=chunk
 ~~~
 
+Chunk numbers are limited to 8 bytes and do not wrap around.  When the consumer gets near the
+end of the sequence number space, it must request a re-keying via the control channel.  Because CCNx in
+a pull-driven model, the consumer is reponsible for the chunk number and thus responsible for requesting
+the re-keying.
```

Comment:
The producer can always request a re-key, but it is the consumer that controls the chunk number space, so its up to the consumer to ensure that there's a rekey before running out of chunk numbers.